app/assets/config/manifest.js

```diff
@@ -1,3 +1,2 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
```

Comment:
Ruby's Sass library cannot yet handle the new syntax. Deleting this line makes it so that our Ruby Sass library stops trying to parse stylesheets.